oidc_cli/rsa-keygen.go

```diff
@@ -0,0 +1,55 @@
+package oidc
+
+import (
+	"fmt"
+	"crypto/rsa"
+	"crypto/rand"
+	"crypto/x509"
+	"golang.org/x/crypto/ssh"
+	"encoding/pem"
+	"gopkg.in/square/go-jose.v2"
+	"os"
+)
+
+func generateRSAKey() error {
```

Comment:
Could you leave a bit of a comment here, something to the effect of:
```
This functions helps generate RSA keys. One of the intended use-cases is to generate keys for the Okta oauth [api authentication](https://developer.okta.com/docs/guides/implement-oauth-for-okta-serviceapp/create-publicprivate-keypair/).  
```
Idk, something like that. Maybe you can come up with a bit better wording